lib/rage/code_loader.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+require "zeitwerk"
+
+class Rage::CodeLoader
+  def initialize
+    @loader = Zeitwerk::Loader.new
+  end
+
+  def setup
+    autoload_path = "#{Rage.root}/app"
+    enable_reloading = Rage.env.development?
+    enable_eager_loading = Rage.env.production?
```

Comment:
What do you think about changing the condition here?

```suggestion
    enable_eager_loading = !Rage.env.development? && !Rage.env.test?
```

Reply:
Totally makes sense!